client/reader/stream/index.jsx

```diff
@@ -60,10 +61,56 @@ function refreshCardFactory( post ) {
 
 const defaultCardFactory = config.isEnabled( 'reader/refresh/stream' ) ? refreshCardFactory : oldCardFactory;
 
+const MIN_DISTANCE_BETWEEN_RECS = 4; // page size is 7, so one in the middle of every page and one on page boundries, sometimes
+const MAX_DISTANCE_BETWEEN_RECS = 49; // page size is 7, so 7 pages
+const RECS_PER_BLOCK = 2;
+
+function getDistanceBetweenRecs() {
+	// the distance between recs changes based on how many subscriptions the user has.
+	// We cap it at MAX_DISTANCE_BETWEEN_RECS.
+	// It grows at the natural log of the number of subs, times a multiplier, offset by a constant.
+	// This lets the distance between recs grow quickly as you add subs early on, and slow down as you
+	// become a common user of the reader.
+	const totalSubs = FeedSubscriptionStore.getTotalSubscriptions();
```

Comment:
heres what it looks like on a [wolfram alpha graph](http://tinyurl.com/h3gamtq
)